python/cuml/dask/ensemble/base.py

```diff
@@ -42,6 +42,7 @@ def _create_model(self, model_func,
         self.client = get_client(client)
         self.workers = self.client.scheduler_info()['workers'].keys()
         self._set_internal_model(None)
+        self.active_workers = list()

```

Comment:
If we set things up expecting 4 workers (len(workers) == 4) and looking for `n_estimators == 400` then we'll expect 100 estimators per worker. But if we only get 3 active workers and just scope down to that, I think we'll only generate 300 total estimators, right?

Reply:
Yes, while collecting the different RF models from workers only those that have data and have fitted a model, will return their models and essentially having just 300 estimators in this case. 

This is used while concatenating the treelite models, not for any other computation - the model params/summary will still reflect correctly.

Reply:
it would be nice to have a warning thrown as well when this happens. Like a simple check to see if the length of `self.active_workers` is the same as `self.workers`. Informing the user when the number of trees the final model has is not the one passed in by them would be nice as well

Reply:
Makes sense! I've added it!

Reply:
I worry this behavior will be very confusing. We train fewer than expected estimators than the user expected, but all of the parameters still look like they have the original number. I think the optimal solution would reallocate the number of estimators per worker to recreate the expected number of estimators.